docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -1,9 +1,69 @@
 
 # Type conversion functions
 
-## toUInt8, toUInt16, toUInt32, toUInt64
+## toInt(8|16|32|64)
 
-## toInt8, toInt16, toInt32, toInt64
+Converts an input value to the [Int](../../data_types/int_uint.md) data type. This functions family includes:
+
+* `toInt8(expr)` — Results in `Int8` data type.
+* `toInt16(expr)` — Results in `Int16` data type.
+* `toInt32(expr)` — Results in `Int32` data type.
+* `toInt64(expr)` — Results in `Int64` data type.
+
+**Parameters**
+
+- `expr` — [Expression](../syntax.md#syntax-expressions) returning a number or a string.
+
+**Returned value**
+
+Integer value in `Int8`, `Int16`, `Int32` or `Int64` data type.
+
+Functions use [rounding towards zero](https://en.wikipedia.org/wiki/Rounding#Rounding_towards_zero), they truncates fraction digits of numbers with floating point.
```

Comment:
what about Decimal?

Reply:
Decimal is a number with floating point, isn't it?

Reply:
no, it's fixed point